attrd/main.c

```diff
@@ -90,6 +96,40 @@ attrd_cpg_destroy(gpointer unused)
 }
 
 static void
+lrm_op_callback(lrmd_event_data_t * op)
+{
+    const char *nodename = NULL;
+
+    CRM_CHECK(op != NULL, return);
+
+    nodename = op->remote_nodename ? op->remote_nodename : attrd_cluster->uname;
+
+    if (op->type == lrmd_event_disconnect && (safe_str_eq(nodename, attrd_cluster->uname))) {
+        crm_err("Lost connection to LRMD service!");
+        attrd_error = ECONNRESET;
+        attrd_shutdown(0);
+        return;
```

Comment:
I thought about the connection method that you pointed out, too.
When the warning transmission was necessary, I make modifications for a method to connect with lrmd.